### ISSUE TITLE
Example of RAABBVI for a Stan model?

### ISSUE THREAD

**Issue by martiningram** [jhuggins/viabel#47]
Hi all,

I enjoyed reading the RAABBVI paper and would like to give the method a go, and also compare it against some other methods. I am wondering what the best way to run it would be, when using a Stan model? I noticed there is a `bbvi` function (https://github.com/jhuggins/viabel/blob/master/viabel/convenience.py#L14). This works fine for me (provided I use pystan 2.19, which is no problem), but it doesn't yet support RAABBVI, is that right? My apologies if I missed an example somewhere.

Thanks for your help,
Best wishes,
Martin

**Comment by jhuggins** [comment]
You are absolutely right. We should definitely add support for RAABBVI to the `bbvi` function @Manushi22

**Comment by martiningram** [comment]
Dear Manushi,

Thank you very much for your reply on this! Not to be impatient (sorry!), but I was just wondering if there was an update on this? It looks like you've added a pull request -- I'd be happy to give that a go, would that be a good idea?

Thank you very much for your help,
All the best,
Martin

**Comment by Manushi22** [comment]
Dear Martin,

Yes, I add `RAABBVI` to `bbvi` function in the pull request. Sorry I did not mention it here. You can try it now. 
Feel free to let me know if you face any problems with the code.

Best,
Manushi

**Comment by martiningram** [comment]
Hi again Manushi,

Thank you, I believe it is working well! Just one question: am I right in assuming that `bbvi` now runs RAABBVI by default, or would I have to give it any special arguments to make sure?

Thanks again and best wishes,
Martin

**Comment by Manushi22** [comment]
Hi Martin,

Now, `bbvi` runs `RAABBVI` by default with default tuning-parameter values. You can always change the tuning-parameter values in `bbvi`.
